src/apps/loan-list/materials/stackable-material/material-info.tsx

```diff
@@ -39,12 +39,16 @@ const MaterialInfo: FC<MaterialInfoProps> = ({
         <div className="list-reservation__about">
           <h3 className="text-header-h4">{title}</h3>
           <p className="text-small-caption color-secondary-gray">
-            <div>
-              {authors && authors} {year && <>({year})</>}
-            </div>
-            <div>{periodical && periodical}</div>
-            <div>{series && series}</div>
+            {authors && authors} {year && <>({year})</>}
```

Comment:
We have the same rendering in other components. I think both in search results and on the material page.
Could we place a TODO here, that we could make a common component for that?

Reply:
Which components?